pandas/tests/internals/test_external_block.py

```diff
@@ -5,12 +5,12 @@
 
 import pandas as pd
 from pandas.core.internals import (
```

Comment:
ideally some more tests

Reply:
Many more tests in the followup PR, coming as soon as this is merged: https://github.com/TomAugspurger/pandas/compare/pandas-array-interface-3...TomAugspurger:pandas-array-upstream+interval#diff-1837f9e869f48877d9a16e2901942b60

Anything in particular you want tested at the block level? Most of those are at the NDFrame level.